src/client/components/ConsentsNavigation.tsx

```diff
@@ -23,13 +24,16 @@ interface ConsentsNavigationProps {
 export const ConsentsNavigation = ({
   showContinueButton = true,
 }: ConsentsNavigationProps) => {
+  const _cmpConsentState = useCmpConsent();
```

Comment:
Curious as to why you are choosing to prefix the variable name with an underscore? (No strong opinion here, just interested in coding style)

Reply:
This could perhaps be `__isC`mpConsentState` since it is representing a boolean here

Reply:
Just following the convention in the code to underscore hidden form values submitted, eg csrfToken.  I think it will help clarify what value is coming in from the form vs the `isCmpConsented` value on the server. 

This draft is very out of date as no  longer using query params.